rollup.types.config.js

```diff
@@ -0,0 +1,36 @@
+import dts from 'rollup-plugin-dts';
+
+export default [
+  {
+    input: 'svelte-package/lib/widget/inline.svelte.d.ts',
+    external: ['./main.css'],
```

Comment:
Is this so that Rollup doesn't try to find or include the types for the CSS file?

Reply:
This was because the rollup was erroring on css files, however this is probabaly due to the fact that it was mapping to the actual js files. i can try to remove this now that we solved that problem.